src/main/java/pipeline_wrapper/Wrapper.java

```diff
@@ -0,0 +1,36 @@
+package pipeline_wrapper;
+
+import java.io.FileInputStream;
+
+import org.hl7.fhir.r4.model.Bundle;
+import org.openhealthtools.mdht.uml.cda.consol.ConsolPackage;
+import org.openhealthtools.mdht.uml.cda.consol.ContinuityOfCareDocument;
+import org.openhealthtools.mdht.uml.cda.util.CDAUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import tr.com.srdc.cda2fhir.transform.CCDTransformerImpl;
+import tr.com.srdc.cda2fhir.util.FHIRUtil;
+import tr.com.srdc.cda2fhir.util.IdGeneratorEnum;
+
+public class Wrapper {
+
+	public static void main(String[] args) throws Exception {
+		final Logger logger = LoggerFactory.getLogger(Wrapper.class);
+		if (args.length > 1) {
+			CDAUtil.loadPackages();
+	
+			FileInputStream fis = new FileInputStream("src/test/resources/" + args[0]);
+			ContinuityOfCareDocument cda = (ContinuityOfCareDocument) CDAUtil.loadAs(fis,
```

Comment:
how so?

Reply:
src/test/resources refer to a local filename inside the jar I believe. And you cannot pass this method to an arbitrary path since we are always adding "src/test/resources/". Or maybe I am missing the reason for this method?

Reply:
The method will be used to establish a pipeline which will feed cda files to this method and will produce fhir json. This method is probably subject to change once we start connecting the pipeline

Reply:
I updated an it works externally. just make sure you feed it with full file path